mirrord/operator/src/client.rs

```diff
@@ -747,6 +654,12 @@ impl OperatorApi<PreparedClientCert> {
     ///
     /// `copy_target` feature is not available for all target types.
     /// Target type compatibility is checked by the operator.
+    // TODO(alex) [high] 2: Now I need to fix this somehow. Ideally, it would make a request
+    // to the operator to create the copy, then we would list the targets and turn it into
+    // a `ResolvedTarget`, or maybe `impl From<CopyTargetCrd> for ResolvedTarget`, instead
+    // of listing.
+    // I think we could come here after `ResolvedTarget` has been used, which is not ideal,
+    // as there will be no target to resolve?
```

Comment:
Not sure I understand you, but this sounds like a merge blocker, no?

Can't you just not resolve the target before calling this, and leave the whole flow as it was before this change?

Reply:
> Can't you just not resolve the target before calling this, and leave the whole flow as it was before this change?

Yup, that's kinda what's in the code now. It was an outdated comment from the `ResolvedTarget::new` again.

Resolving a target should be harmless to the copy target stuff (as far as I'm aware), if we don't check anything in it.

Reply:
And to clarify, we have to resolve a target before anything `copy_target` related, to check if it's an empty deployment.